tests/integrations/test_dbm.py

```diff
@@ -113,3 +118,169 @@ def test_trace_payload_full(self):
                 self._assert_span_is_untagged(span)
             else:
                 self._assert_span_is_tagged(span)
+
+
+# @features.datastreams_monitoring_support_for_kafka
+# @scenarios.integrations
+class _Test_Dbm_Comment:
+    """ Verify DBM comment for given integration """
+
+    integration = None
+    operation = None
+    operation_batch = None
+    execute_batch = False
```

Comment:
Can you 

* add a class `_Test_Dbm_Comment_With_Batch` that inherits from `_Test_Dbm_Comment` and that ships `test_dbm_comment_batch`
* than makes the actual test classes inherits from `_Test_Dbm_Comment_With_Batch`, or `_Test_Dbm_Comment`, depending of which use case they are ? 

The issue with the current implementation is that `test_dbm_comment_batch` will be a success when `execute_batch=False`, and causes some XPASS on libraries that declare the entire class as bug/missing feature.